### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by meomancer**
- [x] show limit size alert for wcs 
- [x] added loading overlay
- [x] change navbar url to ortho_images

### CODE REVIEW

File: django_project/clip-and-ship/views.py [hunk 1]
```diff
@@ -12,6 +12,62 @@
...
 from geonode.layers.views import _resolve_layer, _PERMISSION_MSG_VIEW
 
+WCS_URL = settings.GEOSERVER_BASE_URL + \
+          'wcs?request=getcoverage&' \
+          'version=1.0.0&service=WCS&coverage={layer_name}&' \
+          'format=geotiff&crs=EPSG:4326&bbox={bbox}&width={width}&' \
+          'height={height}'
+
+MAX_CLIP_SIZE = settings.MAXIMUM_CLIP_SIZE
+SIZE_SAMPLE = 10
```
Comment (cchristelis) [inline django_project/clip-and-ship/views.py]: we should make this number a lot higher. I'm thinking in the order of 100

**Comment by cchristelis** [comment]
Hi @meomancer, nice work. I think we should be much more aggressive with the SIZE_SAMPLE setting, I would recommend at least 100 or more, since this is really just to get an estimate and it could still be big if BB is big. If we are limiting in MB perhaps we should make that number around 1000, which would mean a byte for each MB...

**Comment by meomancer** [comment]
hi @cchristelis 
updated, increase size sample to 100
selection_027
selection_029

**Comment by cchristelis** [comment]
On the whole this looks good @ppasq could you test this out on your local?

**Comment by meomancer** [comment]
and also i added loading overlay
selection_028

the loading will be shown when clipping process is run

File: django_project/clip-and-ship/views.py [hunk 2]
```diff
@@ -79,24 +130,51 @@ def clip_layer(request, layername):
...
+        width_sample = int(width / SIZE_SAMPLE)
+        height_sample = int(height / SIZE_SAMPLE)
 
-        response = urllib2.urlopen(wcs_formatted_url)
+        sample_filepath = os.path.join(
+            temporary_folder,
+            layer.title + '.sample.' + extention)
+        download_wcs(
+            layername, bbox_string,
+            SIZE_SAMPLE,
```
Comment (cchristelis) [inline django_project/clip-and-ship/views.py]: shouldn't you be using width_sample here?

File: django_project/clip-and-ship/views.py [hunk 3]
```diff
@@ -79,24 +130,51 @@ def clip_layer(request, layername):
...
+            SIZE_SAMPLE,
+            sample_filepath)
+
+        clipped_size = os.path.getsize(sample_filepath)
+        # size of 10x10 times width_sample * height_sample
+        clipped_size = clipped_size * (width_sample * height_sample)
```
Comment (cchristelis) [inline django_project/clip-and-ship/views.py]: at this point it isn't the clipped_size anymore, let's rather rename this to 'expected_clip_size' or similar.

File: django_project/clip-and-ship/views.py (hunk 2 above)
Comment (meomancer) [reply django_project/clip-and-ship/views.py]: hi @cchristelis 
the idea is size simple is the sample of size, which are 100*100 in the setting
width_sample is the how many width for 100*100 for giving actual width
i think it is naming, i will update it

**Comment by cchristelis** [comment]
very nice work @meomancer

**Pull request merged**